### ISSUE TITLE
Stable version?

### ISSUE THREAD

**Issue by ponasromas** [TheHostingTool/TheHostingTool#146]
Hello,

is it possible to expect stable rewritten version of THT? Such amazing peace of software!

**Comment by rampmaster** [comment]
Good idea +1

**Comment by kmark** [comment]
While I would enjoy doing a complete rewrite of THT it's not really possible for me in the foreseeable future for a few different reasons:

* :watch: Time. Would take quite awhile to match the current number of features THT currently offers (and do them better).
* :dollar: Money. Not hosting or maintenance costs but purely from a time = money standpoint.
* :tada: Interest. Would enough people even use or be in the market for a piece of software like THT? Competition is incredibly stiff when you include the likes of WHMCS, which is directly comparable to THT, and does it for just $15/mo.

**Comment by Trax2k** [comment]
THT should solely be aimed at free hosting providers, there's no other alternative when it comes to post 2 host and i'm sure it could be turned into a decent modern panel for freehosting/p2h etc... I'm also certain that if you charged a license fee for a couple of $$ you would see a small return.

If only i had a clue where to start then i would have been more than to get involved with THT :)

**Comment by kmark** [comment]
> THT should solely be aimed at free hosting providers

Agreed. Those are the only organizations that might find $15/mo painful. I would not have a problem with a "paid plugin" (and THT becoming a de-facto paid hosting tool) but that would not be a part of the core software.

> Post2Host

I've been out of the p2h game for quite some time now. Is the "independent" free/ad-supported/p2h hosting community nearly as big as it was 5 years ago?

> I'm also certain that if you charged a license fee for a couple of $$ you would see a small return.

While a subscription-donation option would be possible, as long as I am involved the software will always be FOSS compliant. I would include in that the absence of nagware or any kind of non-free licensing scheme. To clarify, I'm not looking to make a profit or even see money I spend/spent recouped but rather I cannot at this point personally afford to spend a significant portion of time on something that does not generate revenue.

**Comment by anaszaman** [comment]
Hay @kmark , we already discuss it before. Here is for everyone to check awesome theme for THT http://account.forfity.com

**Comment by xaoseric** [comment]
@kmark i'm working on an a rewrite using laravel 5. Would you be able to transfer ownership of the project?

**Comment by kmark** [comment]
@xaoseric I'd recommend starting from the ground up and re-evaluating what's really needed. At that point it'd be a completely separate piece of software. A new name might be advantageous as it carries none of the baggage THT does. If you developed a compelling THT alternative with a GPL-compatible license, I'd be happy to recommend it to all existing (hopefully none) THT users.

**Comment by xaoseric** [comment]
@kmark THT 2.0 essentially would be built from the ground up. I feel like I can revive the project, it has a legacy of being a good alternative to whmcs and being good for post2host hosts, and I want to revive it. That being said, would an MIT license work for THT 2.0?
